### ISSUE TITLE
Can't filter by Dictionary Product.GetAll

### ISSUE THREAD

**Issue by chrisvell88** [XiaoFaye/WooCommerce.NET#586]
Hi, 

I am using woocommerce.NET library to try and get Product by Purchase Note using the following code however, it seems it is kno recognizing my dictionary filter. Is there a way to get a product by attribute or by purchase not? or is there a list of supported Dictionary fiters?

 Dictionary<string, string> pDic = new Dictionary<string, string>();
 pDic.Add("purchase_note", txtBarcode.Text);
                        App.wooProducts = await wc.Product.GetAll(pDic);

**Comment by XiaoFaye** [comment]
I don't think this is supported by WooCommerce, please check  all available parameters: https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-products

**Comment by chrisvell88** [comment]
Thanks for your reply. From the parameter list it seems you can search by Attribute and Attribute term. So lets say i have an attribute called Barcode and has Term 12345. How can I code that in the dictionary please? can you show me an example?

Thanks

**Comment by chrisvell88** [comment]
Any updates on this please? Need an urgent response
